about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/main.css" >
+    <link rel="stylesheet" href="./styles/nav.css">
+    <link rel="stylesheet" href="./styles/home/intro.css">
+    <link rel="stylesheet" href="./styles/about/intro.css">
+    <link rel="stylesheet" href="./styles/about/logo.css">
+    <link rel="stylesheet" href="./styles/about/prevCamp.css">
+    <link rel="stylesheet" href="./styles/about/partners.css">
+    <title>About Us</title>
+</head>
+
+<body>
+    <header>
+        <nav class="menu-secondary">
+            <ul class="seconday-list">
+                <li><a href="https://www.facebook.com/profile.php?id=100005045626812" target="_blank" rel="noopener"><img
+                            src="./images/social icons/facebook.svg" alt="facebook"></a></li>
+                <li><a href="https://twitter.com/ahmedeid2684" target="_blank" rel="noopener"><img
+                            src="./images/social icons/twitter.svg" alt="twitter"></a></li>
+                <li>English</li>
+                <li>My page</li>
+                <li>Logout</li>
+            </ul>
+        </nav>
+        <nav class="menu-main">
+            <img src="./images/logos/icon-only.png">
```

Comment:
- [ ] Please 🙏🏽 use the `alt` attribute for the image for the best practices of HTML & CSS.